src/extension.ts

```diff
@@ -120,6 +119,10 @@ class ExtensionManager implements vscode.Disposable {
   private readonly _projectOutlineDisposer
       = vscode.window.registerTreeDataProvider('cmake.outline', this._projectOutlineProvider);
 
+  private readonly _configProvider = new CppConfigurationProvider();
+  private _cppToolsAPI?: cpt.CppToolsApi;
```

Comment:
This object is disposable but I don't see it being disposed anywhere or added to the extension context's "subscriptions".

Reply:
I missed that one.